app/scripts/proactive/view/WorkflowView.js

```diff
@@ -518,6 +517,19 @@ define(
             var StudioApp = require('StudioApp');
             var jobModel = StudioApp.models.jobModel;
             var newTaskViews = {};
+            var minLeft = 0;
+            var minTop = 0;
+            function sort(arr, type){
+                return Array.prototype.slice.call(arr).sort(function(a,b){
+                    return a[type] - b[type];
+                })
+            }
+            console.log(tasksPosition)
+            minLeft = sort(tasksPosition, 'left')[0].left;
+            minTop = sort(tasksPosition, 'top')[0].top;
```

Comment:
are all these logs necessary for debugging? Can you keep the essential part of it ?

Reply:
done

Reply:
No, I'll delete them!